app/main.py

```diff
@@ -124,3 +145,51 @@ async def login(form_data: OAuth2PasswordRequestForm = Depends()):
         "access_token": create_access_token(user['username']),
         "refresh_token": create_refresh_token(user['username']),
     }
+
+
+
+@app.get("/downloader_utility_data")
+async def index(taxonomyFilter: str, data_status: str, experiment_type: str):
+    neofourJ = NeoFourJ("bolt://45.88.80.141:30087", "neo4j", "DtolNeo4jAdminUser@123")
```

Comment:
variable in function should be lowercase

Reply:
Resolved